### ISSUE TITLE
RC Test: Ratio and Proportion 1.2.0-rc.1

### ISSUE THREAD

**Issue by zepumph** [phetsims/qa#852]
<h1>RC Test</h1>

**Mentions**: @emily-phet @terracoda @BLFiedler @jessegreenberg 

<h3>Simulation links</h3>

- [github repository for issues](https://github.com/phetsims/ratio-and-proportion/issues)
- [phet top level directory](https://phet-dev.colorado.edu/html/ratio-and-proportion/1.2.0-rc.1)
- [sim: all_phet.html](https://phet-dev.colorado.edu/html/ratio-and-proportion/1.2.0-rc.1/phet/ratio-and-proportion_all_phet.html)

<h3>Test Matrix</h3>

- [General Test Matrix](https://docs.google.com/spreadsheets/d/1UhMTOsOo4byrybtKiaRQWTXi1Zkdty_X-lHfEp8j3Pk/edit#gid=654303995)

<h3>Features included</h3>

- [x] Alternative Input
- [x] UI Sound
- [x] Sonification
- [x] Description
- [x] Voicing
- [x] Camera input (please see https://github.com/phetsims/ratio-and-proportion/issues/488)

<h3>Focus and Special Instructions</h3>

Please note that we have dropped PhET-iO support for this release. Important features for testing are camera input: hands, and voicing, and especially how they interact together.

---
<h2>For QA...</h2>

<details>
<summary><b>General features</b></summary>

<h3>What to Test</h3>

- Click every single button.
- Test all possible forms of input.
  - Test all mouse/trackpad inputs.
  - Test all touchscreen inputs.
- If there is sound, make sure it works.
- Make sure you can't lose anything.
- Play with the sim normally.
- Try to break the sim.
- Test all query parameters on all platforms. (See [QA Book](https://github.com/phetsims/QA/blob/master/documentation/qa-book.md) for a list of query parameters.)
- Download HTML on Chrome and iOS.
- Make sure the iFrame version of the simulation is working as intended on all platforms.
- Make sure the XHTML version of the simulation is working as intended on all platforms.
- Complete the test matrix.
- Don't forget to make sure the sim works with Legends of Learning.
- Test the Game Up harness on at least one platform.
- Check [this](https://docs.google.com/spreadsheets/d/1umIAmhn89WN1nzcHKhYJcv-n3Oj6ps1wITc-CjWYytE/edit#gid=0) LoL
  spreadsheet and notify AR or AM if it not there.
- If this is rc.1 please do a memory test.
- When making an issue, check to see if it was in a previously published version.
- Try to include version numbers for browsers.
- If there is a console available, check for errors and include them in the Problem Description.
- As an RC begins and ends, check the sim repo. If there is a maintenance issue, check it and notify developers if there is a problem.
- As the RC ends, please review the QA credits in the about dialog.  If the QA credits are complete and correct, please add a comment in the RC test issue indicating so.  If not, please open a new issue and specify what changes should be made.

<hr>

</details>

<details>
<summary><b>Accessibility features</b></summary>

<h3>What to Test</h3>

- Specific instructions can be found above.
- Make sure the accessibility (a11y) feature that is being tested doesn't negatively affect the sim in any way. Here is
  a list of features that may be supported in this test:

  - Alternative Input
  - Interactive Description
  - Sound and Sonification
  - Pan and Zoom
  - Mobile Description
  - Voicing
- Test all possible forms of input.
  - Test all mouse/trackpad inputs.
  - Test all touchscreen inputs.
  - Test all keyboard navigation inputs (if applicable).
  - Test all forms of input with a screen reader (if applicable).
- If this sim is not in [this list](https://phet.colorado.edu/en/accessibility/prototypes) or up to date there, make an
  issue in [website](https://github.com/phetsims/website) to ask if PhET research page links need updating. Please
  assign to @terracoda and @emily-phet.

<h3>Screen Readers</h3>

This sim may support screen readers. If you are unfamiliar with screen readers, please ask Katie to introduce you to
screen readers. If you simply need a refresher on screen readers, please consult the
[QA Book](https://github.com/phetsims/QA/blob/master/documentation/qa-book.md), which should have all of the information
you need as well as a link to a screen reader tutorial made by Jesse. Otherwise, look over the a11y view before opening
the simulation. Once you've done that, open the simulation and make sure alerts and descriptions work as intended.

<h3>Platforms and Screen Readers to Be Tested</h3>

- Windows 10 + Latest Chrome + Latest JAWS
- Windows 10 + Latest Firefox + Latest NVDA
- macOS + Safari + VoiceOver
- iOS + Safari + VoiceOver (only if specified in testing issue)

<h3>Critical Screen Reader Information</h3>

We are tracking known screen reader bugs in
[here](https://github.com/phetsims/qa/blob/master/documentation/accessibility-bugs.md). If you find a
screen reader bug, please check it against this list.

<h3>Keyboard Navigation</h3>

This sim supports keyboard navigation. Please make sure it works as intended on all platforms by itself and with a
screen reader.

<h3>Magnification</h3>

This sim supports magnification with pinch and drag gestures on touch screens, keyboard shortcuts, and mouse/wheel controls. Please test magnfication and make sure
it is working as intended and well with the use cases of the simulation. Due to the way screen readers handle user input, magnification is NOT expected to
work while using a screen reader so there is no need to test this case.

<hr>

</details>

<details>
<summary><b>FAQs for QA Members</b></summary>

<br>

  

  <details>
  <summary><i>There are multiple tests in this issue... Which test should I do first?</i></summary> 

Test in order! Test the first thing first, the second thing second, and so on.

  </details>

  <br>

  

  <details>
  <summary><i>How should I format my issue?</i></summary>

Here's a template for making issues:

      <b>Test Device</b>

      blah

      <b>Operating System</b>

      blah

      <b>Browser</b>

      blah

      <b>Problem Description</b>

      blah

      <b>Steps to Reproduce</b>

      blah

      <b>Visuals</b>

      blah

      <details>
      <summary><b>Troubleshooting Information</b></summary>

      blah

      </details>

  </details>

  <br>

  

  <details>
  <summary><i>Who should I assign?</i></summary>

We typically assign the developer who opened the issue in the QA repository.

  </details>

  <br>

  

  <details>
  <summary><i>My question isn't in here... What should I do?</i></summary>

You should:

1. Consult the [QA Book](https://github.com/phetsims/QA/blob/master/documentation/qa-book.md).
2. Google it.
3. Ask Katie.
4. Ask a developer.
5. Google it again.
6. Cry.

  </details>

<br>

<hr>

</details>

**Comment by KatieWoe** [comment]
Memory test
<img width="349" alt="mem" src="https://user-images.githubusercontent.com/41024075/202307041-ce66f650-58a6-4100-ab39-553c0e880993.png">

**Comment by KatieWoe** [comment]
Sounds like a plan. Calling QA here.

**Comment by zepumph** [comment]
Thanks! I think we are pretty close to a new RC, I'm hoping by the end of the week.
